src/builders.ts

```diff
@@ -668,3 +688,63 @@ export class TreeBuilder {
 const structuredClonePolyfill: <T>(value: T) => T = typeof structuredClone === 'function'
   ? structuredClone
   : function (value) { return JSON.parse(JSON.stringify(value)) }
+
+export class LicenseFetcher {
+  readonly LICENSE_FILENAME_PATTERN = /^(?:UN)?LICEN[CS]E|.\.LICEN[CS]E$|^NOTICE$/i
+  readonly LICENSE_FILENAME_BASE = new Set(['licence', 'license'])
+  readonly LICENSE_FILENAME_EXT = new Set([
```

Comment:
dont put this in here. simply create a `_helpers.ts`  file that does this MIME type handling

Reply:
Fixed it.